CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+2.1.1
+------------------
+
+* Messages for `DeserializationException` provide more detail about the involved types when an `IllegalArgumentException` is thrown.
```

Comment:
This should be moved under 2.1.0 as that hasn't been released yet. Also, could you wrap the text at 80 characters?